spec/factories/payment_braintree_transactions.rb

```diff
@@ -5,7 +5,7 @@
     status 0
     amount 12.34
     transaction_created_at "2015-11-17 16:46:19"
-    payment_method_token { "asdf#{Faker::Number.number(10)}" }
-    customer_id { "c#{Faker::Number.number(4)}" }
+    payment_method_id { "asdf#{Faker::Number.number(10)}" }
+    customer_id { Faker::Number.number(4) }
```

Comment:
curious why you removed the letter from the customer id - they can be alphanumeric, that was to trip up any parts of the logic that assumed its an integer